### ISSUE TITLE
How to type double angle quotation marks in Windows/Linux?

### ISSUE THREAD

**Issue by Syntacticlosure** [racket/rhombus#324]
I can't easily type double-angle quotation marks in Win/Linux. I tried \ll and \gg in Drracket, but they are different symbols. M-0171 and M-0187 work in Windows but don't work in Drracket.

A convenient way to type « and » in Win/Linux should exist.

**Comment by rfindler** [comment]
Does Alt Gr+[ and Alt Gr+] work?

**Comment by samth** [comment]
I have mostly used copy and paste on Linux, but on my en-us keyboard layout I don't think AltGr exists.

**Comment by AlexKnauth** [comment]
When I was testing things for helping on [How to Type «…»](https://docs.racket-lang.org/shrubbery/tool-support.html#%28part._type-guillemet%29):

The most convenient way I found on Windows was Alt while numpad 174 for `«`, or Alt while numpad 175 for `»`. The 3-digit 174 and 175 seem easier to remember, especially because the numbers are right next to each other, compared to 0171 and 0187 being different in 2 of their digits.

The most convenient way I found on Linux was using the Compose key. I had to go into the keyboard settings to set the Compose key to something (I happened to pick Right-Alt). But then I could just type Compose `<<` for `«`, or Compose `>>` for `»`.
